core/server/data/schema/versioning.js

```diff
@@ -1,78 +0,0 @@
-var Promise = require('bluebird'),
```

Comment:
This file is now useless, because it was handling getting and setting the database version from settings table.